utils/test-utils/README.md

```diff
@@ -0,0 +1,313 @@
+# Testing-kotlin
+
+In this module you can find the different utilities that we have created in order to help us to implement UI tests on our libraries.
+
+These utilities were originally created as a way to reuse code on our different UI tests without the need to copy-paste useful function on each library module.
+Since we think that there is value on these utilities to be shared with the developer community we have created a testing component that can be added as a external dependency to any project that want to implement UI tests targeting dual-screen and foldable devices.
+
+## Add to your project
+
+1. Make sure you have **mavenCentral()** repository in your top level **build.gradle** file:
+
+    ```gradle
+    allprojects {
+        repositories {
+            google()
+            mavenCentral()
+         }
+    }
+    ```
+
+2. Add dependencies to the module-level **build.gradle** file (current version may be different from what's shown here):
+
+    ```gradle
+    implementation "com.microsoft.device.dualscreen.testing:testing-kotlin:1.0.0-alpha2"
+    ```
+
+3. Optional. If you need to use other testing dependencies such as [Espresso](https://developer.android.com/training/testing/espresso) or [UiAutomator](https://developer.android.com/training/testing/other-components/ui-automator), you will have to add them to your **build.gradle** file as well. This library uses these dependencies but are not exposed.
+
+# API reference
+
+The library is divided into a few classes that cover just one specific purpose:
```

Comment:
What is the purpose?

Reply:
wops! no! it shouldn't be there! good catch!

Reply:
Acts as the entry starting point for the following classes API